### PR TITLE
WIP Cromwell-new-venues

### PR DESCRIPTION
**Pull request by cwcromwell**
### Info

- [x] I have served the site locally and verified that the build is successful and my changes render correctly. 

*   Closes `#1` 

|Info|Yours|
|:---|:---|
|Operating System|`Enter your computer's operating system here`|
|Ruby Version|`Enter your ruby version here`|

### CODE REVIEW

**Comment by cwcromwell** [comment]
I attempted to write down the instructions for creating a new venue, while creating venues. This probably needs some work. (i.e. the new venues don't show up when I expect them to.) I might be missing something about the way the rules work for naming the location and map.

This has at least two reasons to be a WIP: 
* Need to verify the instructions in `contributing.md`
* Need some parking info about Flatiron School (for the venue info)

**Comment by cwcromwell** [comment]
I'm not sure why Gemfile is showing changes. Should this be noted in a .gitignore?

**Comment by clairealyssa** [comment]
> I'm not sure why Gemfile is showing changes. Should this be noted in a .gitignore?

No, we want the Gemfile to be versioned. I'll take a look at what it changed for you and update the logic so that your system won't make those changes again.

File: README.md [hunk 1]
```diff
@@ -21,6 +26,16 @@ To install dependencies:
     ```bash
     bundle install
     ```
+### Build the site
```
Comment (clairealyssa) [inline README.md]: This feels unnecessary? jekyll serve will build the site and create the _site folder output, and we've already linked the Jekyll docs.

**Comment by clairealyssa** [comment]
Merging. Forgot to take the WIP off...and didn't see it until after I'd merged. @cwcromwell Did you have other changes that you were planning to add in this PR? Or was it just the open question?

**Pull request merged**